build.gradle

```diff
@@ -175,7 +175,7 @@ allprojects {
 
 	checkstyle {
 		configFile = rootProject.file("checkstyle.xml")
-		toolVersion = "8.43"
+		toolVersion = "9.1"
```

Comment:
Is there a reason for this update?

Reply:
it was a merge conflict that I resolved. it'd be complaining about it if I didn't, IDK why the former red is showing that it is 8.43 since I was just updating it to match 9.1

https://github.com/FabricMC/fabric/blob/fa34ed2c10c399ed6c973a56f3fda31cd47811f6/build.gradle#L142